CHANGELOG.rst

```diff
@@ -1,6 +1,39 @@
 Change log
 ==========
 
+0.19.0 (2020-08-??)
+-------------------
+
+BREAKING CHANGES
+~~~~~~~~~~~~~~~~
+
+* **migrate to Python3.7**
+* **disable entry points and switch to ``python -m`` calls**
+* **rename ``_universum`` to ``universum``** *(configs need to be fixed)*
+
+New features
+~~~~~~~~~~~~
+
+* **github-handler:** add :doc:`a new Universum mode <args_github_handler>` to serve as a GitHub App
+* **pylint:** allow selecting exact python version
+* **nonci:** set project root to current directory
+* **github_vcs:** aquire GitHub token only when needed
+
+Bug fixes
+~~~~~~~~~
+
+* **swarm:** only update status for latest revisions
+* **nonci:** remove code report from nonci, launching steps twice
```

Comment:
Please make it a proper sentence, for example:
fixed the issue with launching code report steps twice in nonci mode.